solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -396,6 +399,34 @@ library DepositUtils {
         return _postCallBalance.sub(_preCallBalance);
     }
 
+    /// @notice     Adds a given amount to an addresses withdrawal allowance.
```

Comment:
I would rephrase “Adds a given amount to the withdrawal allowance for the address.” to avoid the awkward address possessive here…